insonmnia/dwh/server.go

```diff
@@ -1157,3 +1217,40 @@ func (w *DWH) checkBenchmarks(benches *pb.Benchmarks) error {
 
 	return nil
 }
+
+func (w *DWH) removeEntityID(id *big.Int, entity string) error {
```

Comment:
The name is weird, maybe will be better to use smth like `removeSlateEntityByID`?